run-benchmark: Some plans point to WebKit SVN repository

mrego opened the pull request:
#### dddaa5680add665ca721b5985d4169c520fe45a5
<pre>
run-benchmark: Some plans point to WebKit SVN repository
<a href="https://bugs.webkit.org/show_bug.cgi?id=297391">https://bugs.webkit.org/show_bug.cgi?id=297391</a>

Reviewed by Carlos Alberto Lopez Perez.

Fix plans so they point to the Git repository instead of the SVN one.

* Tools/Scripts/webkitpy/benchmark_runner/data/plans/content-animation.plan:
* Tools/Scripts/webkitpy/benchmark_runner/data/plans/jetstream.plan:
* Tools/Scripts/webkitpy/benchmark_runner/data/plans/jetstream2.0.plan:
* Tools/Scripts/webkitpy/benchmark_runner/data/plans/motionmark1.0.plan:
* Tools/Scripts/webkitpy/benchmark_runner/data/plans/motionmark1.1.plan:
* Tools/Scripts/webkitpy/benchmark_runner/data/plans/speedometer1.0.plan:
* Tools/Scripts/webkitpy/benchmark_runner/data/plans/speedometer2.0.plan:

Canonical link: <a href="https://commits.webkit.org/298817@main">https://commits.webkit.org/298817@main</a>
</pre>
<!--EWS-Status-Bubble-Start-->
https://github.com/WebKit/WebKit/commit/52a65ad7fb56c557dee3d94a1250c9d3d90e0496

| Misc | iOS, visionOS, tvOS & watchOS  | macOS  | Linux |  Windows |
| ----- | ---------------------- | ------- |  ----- |  --------- |
| [✅ 🧪 style](https://ews-build.webkit.org/#/builders/38/builds/116349 "Passed style check") | [✅ 🛠 ios](https://ews-build.webkit.org/#/builders/131/builds/36010 "Built successfully") | [✅ 🛠 mac](https://ews-build.webkit.org/#/builders/138/builds/26560 "Built successfully") | [✅ 🛠 wpe](https://ews-build.webkit.org/#/builders/5/builds/122406 "Built successfully") | [✅ 🛠 win](https://ews-build.webkit.org/#/builders/59/builds/66910 "Built successfully") 
| [✅ 🧪 bindings](https://ews-build.webkit.org/#/builders/9/builds/118238 "Passed tests") | [✅ 🛠 ios-sim](https://ews-build.webkit.org/#/builders/130/builds/36704 "Built successfully") | [✅ 🛠 mac-AS-debug](https://ews-build.webkit.org/#/builders/123/builds/44598 "Built successfully") | [✅ 🧪 wpe-wk2](https://ews-build.webkit.org/#/builders/34/builds/88375 "Passed tests") | [  ~~🧪 win-tests~~](https://ews-build.webkit.org/#/builders/60/builds/42858 "Build was cancelled. Recent messages:Printed configuration") 
| [✅ 🧪 webkitperl](https://ews-build.webkit.org/#/builders/11/builds/119298 "Passed tests") | [✅ 🧪 ios-wk2](https://ews-build.webkit.org/#/builders/132/builds/29276 "Passed tests") | [✅ 🧪 api-mac](https://ews-build.webkit.org/#/builders/18/builds/104390 "Passed tests") | [✅ 🧪 api-wpe](https://ews-build.webkit.org/#/builders/41/builds/68807 "Passed tests") | 
| [✅ 🧪 webkitpy](https://ews-build.webkit.org/#/builders/6/builds/115723 "Passed tests") | [✅ 🧪 ios-wk2-wpt](https://ews-build.webkit.org/#/builders/133/builds/28358 "Passed tests") | [✅ 🧪 mac-wk1](https://ews-build.webkit.org/#/builders/135/builds/22493 "Passed tests") | [✅ 🛠 wpe-cairo](https://ews-build.webkit.org/#/builders/65/builds/66087 "Built successfully") | 
| | [✅ 🧪 api-ios](https://ews-build.webkit.org/#/builders/13/builds/98658 "Passed tests") | [✅ 🧪 mac-wk2](https://ews-build.webkit.org/#/builders/136/builds/22654 "Passed tests") | [✅ 🛠 gtk](https://ews-build.webkit.org/#/builders/2/builds/125555 "Built successfully") | 
| | [✅ 🛠 vision](https://ews-build.webkit.org/#/builders/128/builds/43243 "Built successfully") | [✅ 🧪 mac-AS-debug-wk2](https://ews-build.webkit.org/#/builders/122/builds/32477 "Passed tests") | [✅ 🧪 gtk-wk2](https://ews-build.webkit.org/#/builders/1/builds/97081 "Passed tests") | 
| | [✅ 🛠 vision-sim](https://ews-build.webkit.org/#/builders/121/builds/43608 "Built successfully") | [✅ 🧪 mac-wk2-stress](https://ews-build.webkit.org/#/builders/8/builds/100596 "Passed tests") | [✅ 🧪 api-gtk](https://ews-build.webkit.org/#/builders/21/builds/96877 "Passed tests") | 
| [✅ 🛠 🧪 merge](https://ews-build.webkit.org/#/builders/19/builds/24731 "Built successfully and passed tests") | [✅ 🧪 vision-wk2](https://ews-build.webkit.org/#/builders/126/builds/42166 "Passed tests") | [✅ 🧪 mac-intel-wk2](https://ews-build.webkit.org/#/builders/137/builds/20065 "Passed tests") | [✅ 🛠 playstation](https://ews-build.webkit.org/#/builders/134/builds/39196 "Built successfully") | 
| | [✅ 🛠 tv](https://ews-build.webkit.org/#/builders/127/builds/43130 "Built successfully") | | | 
| | [✅ 🛠 tv-sim](https://ews-build.webkit.org/#/builders/125/builds/42597 "Built successfully") | | | 
| | [✅ 🛠 watch](https://ews-build.webkit.org/#/builders/129/builds/45937 "Built successfully") | | | 
| | [✅ 🛠 watch-sim](https://ews-build.webkit.org/#/builders/124/builds/44302 "Built successfully") | | | 
<!--EWS-Status-Bubble-End-->

clopez reviewed (changes requested):
The patch looks good, but the git hashes are not correct. See comment below

clopez commented on Tools/Scripts/webkitpy/benchmark_runner/data/plans/content-animation.plan:
```diff
@@ -1,7 +1,8 @@
     "count": 5,
-    "svn_source": "https://svn.webkit.org/repository/webkit/trunk/PerformanceTests/Animation/@r205680",
+    "local_git_archive": "PerformanceTests/Animation@284845675d6fe730f5a7630e3185fc417a0e8781",
+    "github_source": "https://github.com/WebKit/WebKit/tree/284845675d6fe730f5a7630e3185fc417a0e8781/PerformanceTests/Animation",
```
This is 908fbd47ddf27b975b278b1d6d22cc3c0e4ec8b7

mrego replied:
Done.

clopez commented on Tools/Scripts/webkitpy/benchmark_runner/data/plans/jetstream.plan:
```diff
@@ -1,7 +1,8 @@
     "count": 5,
-    "svn_source": "https://svn.webkit.org/repository/webkit/trunk/PerformanceTests/JetStream/@r190897",
+    "local_git_archive": "PerformanceTests/JetStream@8a40dff5cfb674cece475d11af5870400dac26e6",
+    "github_source": "https://github.com/WebKit/WebKit/tree/8a40dff5cfb674cece475d11af5870400dac26e6/PerformanceTests/JetStream",
```
This is 3e2f1f4881d91cbf488ad87a5bfa9e7f844de06f

mrego replied:
Done.

clopez commented on Tools/Scripts/webkitpy/benchmark_runner/data/plans/jetstream2.0.plan:
```diff
@@ -1,7 +1,8 @@
     "count": 5,
-    "svn_source": "https://svn.webkit.org/repository/webkit/trunk/PerformanceTests/JetStream2/@r286562",
+    "local_git_archive": "PerformanceTests/JetStream2@50012e72f222c458d7b416e06becc1a5e035d336",
+    "github_source": "https://github.com/WebKit/WebKit/tree/50012e72f222c458d7b416e06becc1a5e035d336/PerformanceTests/JetStream2",
```
This is 43136e0015ca5360a88787a0ffa52e9ced7b2481

mrego replied:
Done.

clopez commented on Tools/Scripts/webkitpy/benchmark_runner/data/plans/motionmark1.0.plan:
```diff
@@ -1,7 +1,8 @@
     "count": 1,
-    "svn_source": "https://svn.webkit.org/repository/webkit/trunk/PerformanceTests/MotionMark/@r210459",
+    "local_git_archive": "PerformanceTests/MotionMark@68ca095dc481349bef7e00b97b32fcffa020c119",
+    "github_source": "https://github.com/WebKit/WebKit/tree/68ca095dc481349bef7e00b97b32fcffa020c119/PerformanceTests/MotionMark",
```
This is 119e2d3f1c8f23a32bde610b51b03dd3ab416f6a

mrego replied:
Done.

clopez commented on Tools/Scripts/webkitpy/benchmark_runner/data/plans/motionmark1.1.plan:
```diff
@@ -1,7 +1,8 @@
     "count": 1,
-    "svn_source": "https://svn.webkit.org/repository/webkit/trunk/PerformanceTests/MotionMark/@r278436",
+    "local_git_archive": "PerformanceTests/MotionMark@c864765c693d31b70ece9a455e7b82b413041a6a",
+    "github_source": "https://github.com/WebKit/WebKit/tree/c864765c693d31b70ece9a455e7b82b413041a6a/PerformanceTests/MotionMark",
```
This is 1d7c87a8cc9a4238f3b522fa311192924d91d46a

mrego replied:
Done.

clopez commented on Tools/Scripts/webkitpy/benchmark_runner/data/plans/speedometer1.0.plan:
```diff
@@ -1,7 +1,8 @@
     "count": 5,
-    "svn_source": "https://svn.webkit.org/repository/webkit/trunk/PerformanceTests/Speedometer/@r183695",
+    "local_git_archive": "PerformanceTests/Speedometer@5ce8e1050c4d1ae300524c08eb4a83f6fdda0a88",
+    "github_source": "https://github.com/WebKit/WebKit/tree/5ce8e1050c4d1ae300524c08eb4a83f6fdda0a88/PerformanceTests/Speedometer",
```
This is 4a78b14015a1ea8545d964a8a0a3c57e5b0bf869

mrego replied:
Done.

clopez commented on Tools/Scripts/webkitpy/benchmark_runner/data/plans/speedometer2.0.plan:
```diff
@@ -1,7 +1,8 @@
     "count": 4,
-    "svn_source": "https://svn.webkit.org/repository/webkit/trunk/PerformanceTests/Speedometer/@r226694",
+    "local_git_archive": "PerformanceTests/Speedometer@f4a1928971b4681bbfa351de32fb05adcdb35d1f",
+    "github_source": "https://github.com/WebKit/WebKit/tree/f4a1928971b4681bbfa351de32fb05adcdb35d1f/PerformanceTests/Speedometer",
```
This is 62c9a5959d1121559ebc4dfe18d1fa275748b49d

mrego replied:
Done.

clopez reviewed (commented):
The old revision numbers are not the same than the new canonical link ids, the numbers are off by thousands of commits.

To look for those you can use this:

```
git log --grep trunk@183695
```

where `183695` is the numeric part of `@r183695`

clopez reviewed (approved):
LGTM

Thanks!

mrego committed:
run-benchmark: Some plans point to WebKit SVN repository
https://bugs.webkit.org/show_bug.cgi?id=297391

Reviewed by Carlos Alberto Lopez Perez.

Fix plans so they point to the Git repository instead of the SVN one.

* Tools/Scripts/webkitpy/benchmark_runner/data/plans/content-animation.plan:
* Tools/Scripts/webkitpy/benchmark_runner/data/plans/jetstream.plan:
* Tools/Scripts/webkitpy/benchmark_runner/data/plans/jetstream2.0.plan:
* Tools/Scripts/webkitpy/benchmark_runner/data/plans/motionmark1.0.plan:
* Tools/Scripts/webkitpy/benchmark_runner/data/plans/motionmark1.1.plan:
* Tools/Scripts/webkitpy/benchmark_runner/data/plans/speedometer1.0.plan:
* Tools/Scripts/webkitpy/benchmark_runner/data/plans/speedometer2.0.plan:

Canonical link: https://commits.webkit.org/298817@main
